Hotails/urls.py

```diff
@@ -19,5 +19,6 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', views.homepage, name='homepage')
+    path('', views.homepage, name='homepage'),
+    path('about/', views.about, name='about'),
```

Comment:
I think best will be to add all pages to the urls and direct all to the homepage view